scribble/app/views/posts/edit.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Editing Post</h1>
+
+<%= render 'form' %>
+
+<%= link_to 'Back', posts_path(post) %>
```

Comment:
`<%= link_to "Back", post_path(@post) %>`
This would send us back to the show view for the post we are editing as long as `@post` is defined.

Otherwise you could use `posts_path` to go back to the index view, note `posts_path` does not need an argument.